components/Main/Exercises/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,123 @@
+import styled from "styled-components";
+
+export default function SearchBar({ onChange }) {
+  function handleSearchClick() {
+    if (!document.getElementById("search-bar").focus()) {
+      document.getElementById("search-bar").blur();
+    }
+  }
+
+  return (
+    <StyledSearchBar>
+      <input
+        type="text"
+        id="search-bar"
+        placeholder="Search..."
+        onChange={onChange}
+      />
+      <span onClick={handleSearchClick}></span>
+    </StyledSearchBar>
+  );
+}
+
+export const StyledSearchBar = styled.div`
```

Comment:
You have a styled component as a JS file, so just import it here and use it exactly the same :)